### PR TITLE
New clientcount download

### PR DESCRIPTION
**Pull request by alex-silverman**

### CODE REVIEW

**Review by kiadso — CHANGES_REQUESTED** [review]
These numbers are looking good. In the card, I added a spot for the 3rd tab that outlines which columns are needed there. I'm going to list a few small things that need attention before merge:

- [ ]  For each tab, column order to align with the way the card has it
- [ ]  The Date Range tab is missing the Exit-related columns
- [ ]  Added some arrange() code to the card 

I really like the workbook with the 3 tabs in it. I know we talked about where to put the button since the "Select Project" doesn't affect its results. The date range does affect the results. So I'm wondering if we move the Select Project to its own box named Project-level or something and put it below the box that already exists? We can make it that same color as the other project-level boxes..

File: 01_cohorts.R
Comment (alex-silverman) [reply 01_cohorts.R]: i think we decided to remove this variable. I'm not sure why you were seeing it. My github shows maybe you were looking at an outdated version?

**Comment by alex-silverman** [comment]
> These numbers are looking good. In the card, I added a spot for the 3rd tab that outlines which columns are needed there. I'm going to list a few small things that need attention before merge:
> 
> * [ ]   For each tab, column order to align with the way the card has it
> * [ ]   The Date Range tab is missing the Exit-related columns
> * [ ]   Added some arrange() code to the card
> 
> I really like the workbook with the 3 tabs in it. I know we talked about where to put the button since the "Select Project" doesn't affect its results. The date range does affect the results. So I'm wondering if we move the Select Project to its own box named Project-level or something and put it below the box that already exists? We can make it that same color as the other project-level boxes..



> These numbers are looking good. In the card, I added a spot for the 3rd tab that outlines which columns are needed there. I'm going to list a few small things that need attention before merge:
> 
> * [ ]   For each tab, column order to align with the way the card has it
> * [ ]   The Date Range tab is missing the Exit-related columns
> * [ ]   Added some arrange() code to the card
> 
> I really like the workbook with the 3 tabs in it. I know we talked about where to put the button since the "Select Project" doesn't affect its results. The date range does affect the results. So I'm wondering if we move the Select Project to its own box named Project-level or something and put it below the box that already exists? We can make it that same color as the other project-level boxes..

1/15/23 AS: I'm not seeing the 3rd tab in the card. But, for consistency, shouldn't the 3rd tab - the details tab - just include the same columns as the Details table in the app?

**Comment by alex-silverman** [comment]
I think this is just waiting on your approval now, @kiadso!

**Pull request merged**
